ambry-store/src/test/java/com.github.ambry.store/IndexSegmentTest.java

```diff
@@ -219,10 +261,12 @@ private IndexSegment createIndexSegmentFromFile(File file, boolean isMapped, Jou
    * @param segment the {@link IndexSegment} to add the entries to.
    * @param referenceIndex the {@link NavigableMap} to add all the entries to. This repreents the source of truth for
    *                       all checks.
+   * @return {@link SortedMap} of MockIds to {@link IndexValue}s that were added to the {@link IndexSegment}
    * @throws StoreException
    */
-  private void addPutEntries(List<Long> offsets, long lastEntrySize, IndexSegment segment,
+  private ArrayList<IndexEntry> addPutEntries(List<Long> offsets, long lastEntrySize, IndexSegment segment,
```

Comment:
`List` - use interfaces unless you require specific functions of the implementations (you don't here)